ex03.ex

```diff
@@ -104,8 +108,9 @@ defmodule Ex03 do
       false
 
   """
-
-  def list_equal . . . "your code"
+  def list_equal([], []), do: true
```

Comment:
# -1

I'd make the same comment about using pattern matching:

~~~ elixir
  def list_equal([], []),               do: true
  def list_equal([ h | t1 ], [h | t2]), do: list_equal(t1, t2)
  def list_equal(_, _),                 do: false
~~~